db/migrate/20160302203922_remove_is_datacenter_from_ems_folder.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveIsDatacenterFromEmsFolder < ActiveRecord::Migration[5.0]
+  def up
+    # Go through all existing EmsFolder records and set :type => Datacenter
+    # if the is_datacenter column is true before deleting it
+    EmsFolder.all.each do |folder|
```

Comment:
Note, we want the stub here because we don't want to use the app/models/ems_folder.rb to be used as this migration could be run after the model has some backwards incompatible change.